.github/workflows/ci.yml

```
@@ -4,7 +4,8 @@ on:
   push:
     branches: [master]
   pull_request:
-    branches: [master]
+    branches:
+      - '**'
 
 jobs:
   build:
```

```suggestion
  build:
    name: '${{ matrix.testname }}'
```
The name I saw in the run is a bit difficult to parse. You can add the django-env or format it differently, but you get the gist.

---

That would be neat. But I'm not sure it's worth creating a migration issue

---

Won't all this changes fall in to one issue?